runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -104,6 +104,26 @@ void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::Tr
    treetop->insertAfter(TR::TreeTop::create(comp(), TR::Node::create(node, TR::treetop, 1, newCallNode)));
    }
 
+void J9::RecognizedCallTransformer::process_java_lang_StrictMath_sqrt(TR::TreeTop* treetop, TR::Node* node)
+   {
+   if (!node->getSymbol()->castToResolvedMethodSymbol()->canReplaceWithHWInstr())
```

Comment:
Not a fan of this API. We should directly test for `TR::Compiler->target.cpu.getSupportsHardwareSQRT()`. Also having the negation here `!` seems like a bug.